mkdocs/localization.py

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+
+from __future__ import absolute_import, unicode_literals
```

Comment:
This line should be unnessecary as we have since dropped support for the older Python versions that needed it.

Reply:
Ok; as a side note, did you consider moving the code base to `black` and using `pyupgrade` to keep the syntax clean and up to date?

Reply:
We did run `pyupgrade` when dropping Python 2.7 support. Of course, the contents of the PR were not included in that.